src/core/metrics/app-metrics.ts

```diff
@@ -59,34 +64,46 @@ export class AppMetrics {
   }
 
   private constructor() {
-    globalThis.KONTUR_METRICS = {
-      watchList: this.watchList,
-      markers: this.markers,
-      sequences: this.sequences,
-      events: this.eventLog,
-      toggleAlert: () => this.settings.toggle('KONTUR_SQ_ALERT'),
-      toggleLog: () => this.settings.toggle('KONTUR_SQ_LOG'),
-    };
+    //
   }
 
-  init(appId: string, route: string) {
-    console.info('appMetrics.init with', appId, route);
+  init(appId: string, route: string, hasFeature: (f: AppFeatureType) => boolean): void {
     // currently we support metrics only for map page
     if (route !== '') {
       // '' is route for map
       return;
     }
 
     this.reportTemplate.appId = appId ?? '';
+
+    // add available features to metrics
+    this.watchList = buildWatchList(hasFeature);
+
     globalThis.addEventListener(METRICS_EVENT, this.listener.bind(this) as EventListener);
+
     if (KONTUR_METRICS_DEBUG) {
-      console.info('appMetrics.init ready', this.reportTemplate);
+      console.info(`appMetrics.init route:${route}`, this.reportTemplate, this.watchList);
     }
+
+    this.exposeMetrics();
   }
 
   // remove listeners and unsubscribe from atoms
   cleanup() {
     globalThis.removeEventListener(METRICS_EVENT, this.listener as EventListener);
+    this.watch = () => null;
```

Comment:
why are we rewriting method on cleanup?

Reply:
Some "events" still coming from reatom store onPatch handler (that will change in next iteration to normal events). Here we need to stop watch